code/__DEFINES/demo.dm

```diff
@@ -0,0 +1,38 @@
+#define MARK_TURF(T)\
```

Comment:
I fucking knew it tg, I knew you were gonna ruin my beautiful code with more #define macros.

Reply:
"beautiful code"

Reply:
The defines don't want to work so your beautiful code shall remain intact